apprise/Apprise.py

```diff
@@ -386,7 +387,12 @@ def async_notify(self, *args, **kwargs):
         (This is omitted to preserve syntax compatibility with Python 2.)
 
         The arguments are identical to those of Apprise.notify().
+
+        # TODO: Add `async` keyword.
         """
+        warnings.warn("A future version of Apprise will add the `async` "
```

Comment:
The `#TODO` ref is good, but the `async` is [already in place today](https://github.com/caronc/apprise/issues/396#issuecomment-857046777)

We don't need this warning at all.  We just didn't have the `async` keyword there (and had some magic in front of it, so the code could co-exist in a Python v2.7 environment. we're good here for sure.

Reply:
Maybe I am not right on track yet, but didn't we just observe that adding `async` breaks the test suite completely, as it will likewise do for others?

Maybe it is too late at night already for me ;]. Feel free to adjust as necessary. Thanks for your patience!

Reply:
You might be right here; but to my knowledge we were `async` code that under the hood created an `async` object.  The chagne Ryan would make would be more then the bug i introduced by only updating one function.   The wrappers under the hood will disappear .  But i could also be misinterpreting what is wrong here.